### DISCUSSION TITLE
Anonymous Authentication for Easy Filtering

### DISCUSSION THREAD

**Discussion by noob8boi** [pocketbase/pocketbase#5093] [Q&A]
I was going through pocketbase and realised I still want users to be able to view collection records without oauth2/email+pass authentication. Basically like guest but still a session is there. Now for api filter rule I think only letting non empty string auth id access the data means that guests can't access it. Anyway if that's the case I will have to implement my own for the anonymous authentication?

I am still new so might have missed some points.

**Comment by mrvfino** [comment]
I'm exploring this as well. I think it's easier just to create a dummy account for them then just update their password if they want to finally sign in / register :)
